module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -69,14 +76,24 @@ A vignette on the usage of the `ggplot2` package can be found [**here**](https:/
 |--------------------------------------|-----------------------------------------------------|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
 | `ggplot2`         | [`ggplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggplot)                   | GG Plot               | Begins a plot that is finished by adding layers.        |
+| `ggplot2`               | [`xlab()`; `ylab()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/labs)           | X Axis Labels; Y Axis Labels                    | Modifies the labels on the x axis and on the y axis, respectively        |             
 | `ggplot2`         | [`geom_boxplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/geom_boxplot)       | Boxplot               | Creates a boxplot when combined with ggplot()            |
+| `ggplot2`         | [`geom_density()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/geom_density)       | Density Plot               | Creates a smoothed plot when combined with ggplot() based on the computed density estimate            |
+| `ggplot2`         | [`theme_classic()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggtheme)               | Classic Theme   | Displays `ggplot` without gridlines                                                                         |
+| `ggplot2`            | [`facet_wrap()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/facet_wrap)                     | Facet Wrap      | Plots individual graphs using specified variables to subset the data                                                                   |
+| `ggplot2`         | [`ggtitle()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/labs)                              | GG Title        | Sets the title of a `ggplot`                         |
+| `ggplot2`               | [`ggsave()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggsave)                 | GG Save                                         | Saves the last plot in working directory                                 |                                                                                         
+| `ggplot2`               | [`last_plot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/last_plot)           | Last plot                                       | Returns the last plot produced                                           |  
 
 
-### `readr` and `tibble`
+### `readr`, `tibble` and `tidyr`
```

Comment:
You could just call these "Other tidyverse" or something?

Reply:
May just want to include the `ggplot`functions in there and call them all underneath `tidyverse` then have a subsection that's `ggplot`

Reply:
Part of the reason this was done was to link to each individual package documentation and vignette.

Using the suggestion above, I am thinking that we can use `tidyverse` as the big header, and keep the smaller headers (`dplyr`, `ggplot2`, and `readr, tibble and tidyr`) as subsections. What do you think about that idea?

Reply:
Or maybe we can rename the header `readr, tibble and tidyr` to be `Other tidyverse` and link the individual documentations under that heading.